### ISSUE TITLE
Some way to not have the image so occluded with both a gray fade and text

### ISSUE THREAD

**Issue by f-rank** [dfaker/embedding-to-png-script#6]
The text is particularly poorly placed and a bit too big.
It usually shows up on top of the place where faces go
The gray color fade gives it further poor recognition of what is going on under it all.

Can at least the gray be toned down, or simply removed someway, maybe an option for it.
Looking at hundreds of TI embeds that all have a gray pattern over them kind of defeats the point of this.

**Comment by zrichz** [comment]
I agree with OP - the idea of visually seeing what png's are actually embeddings is GREAT, and the edge markings do this. I have no problem with the text at all, but the grey fade is overbearing and tbh defeats the object of visually judging what styles one has available. I realise there needs to be *some* fade so text is readable, but please can the fade perhaps be from top/bottom 20% - Have some way of seeing the image more clearly than is presently the case.
